scss/functions/_get-color.scss

```diff
@@ -7,4 +7,11 @@
   // if they are valid, return the color
   // if they are not valid warn 'Invalid color type or level' and return null
   // null is a special value in Sass that means nothing, and it is used to represent the absence of a value, you need to return null if the type or level is not valid
+  $hexColor: map.get(map.get($colors, $type), $level);
+  @if ($hexColor != null) {
+    @return $hexColor;
+  } @else {
+    @warn "Invalid color type or level";
+  }
+  @return null;
```

Comment:
masih kurang tepat ya saran saya coba manfaatkan `map.has-key` pada `@if` operator

Reply:
Change to
```
  @if (map.has-key($colors, $type, $level)) {
    @return map.get(map.get($colors, $type), $level);
  } @else {
    @warn "Invalid color type or level";
    @return null;
  }
```